hc/accounts/tests/test_profile.py

```diff
@@ -18,8 +18,13 @@ def test_it_sends_set_password_link(self):
         self.alice.profile.refresh_from_db()
         token = self.alice.profile.token
         ### Assert that the token is set
+        assert token 
```

Comment:
This assertion does not conform with the agreed style. You could use
```python
self.assertIsNotNone(token)
```
in this case..